datasets/standvastgoed/dataset.json

```diff
@@ -16,9 +16,9 @@
   "tables": [
     {
       "id": "gebouwen",
-      "$ref": "gebouwen/v1.0.0",
+      "$ref": "gebouwen/v1.1.0",
       "activeVersions": {
-        "1.0.0": "gebouwen/v1.0.0"
+        "1.1.0": "gebouwen/v1.1.0"
```

Comment:
Er worden velden verwijderd dus oude objecten zijn niet meer geldig in het nieuwe schema. Dit moet dus een major versie bump zijn. (naar 2.0.0)